.changeset/chubby-flies-shine.md

```diff
@@ -0,0 +1,7 @@
+---
+"hardhat": patch
+---
+
+Upgraded EDR to [v0.11.1](https://github.com/NomicFoundation/edr/releases/tag/%40nomicfoundation%2Fedr%400.11.1):
+
+- Fixed bug where native token sent to test account does not increase recipient balance.
```

Comment:
This is surprising: apparently changesets doesn't work with `- `, as it uses them to create a changelog.

Also, should we explain the bug a little? What about this?

```suggestion
Upgraded EDR to [v0.11.1](https://github.com/NomicFoundation/edr/releases/tag/%40nomicfoundation%2Fedr%400.11.1), which fixed a bug when sending ETH to the testing accounts in forked networks. Now testing accounts are automatically undelegated.
```